### PR TITLE
feat: change login callback behavior

### PR DESCRIPTION
**Pull request by xiaolang124**
1. Change login callback behavior.
2. Change extension graph log level to error.

### CODE REVIEW

**Review by a1exwang — APPROVED** [review]

**Comment by a1exwang** [comment]
LGTM

File: packages/api/src/utils/login.ts [hunk 1]
```diff
@@ -50,8 +50,9 @@ export interface AzureAccountProvider {
      * Add update account info callback 
      * @param name callback name
      * @param statusChange callback method
+     * @param immediaCall whether callback when register
      */
-    setStatusChangeMap(name: string, statusChange: (status: string, token?: string, accountInfo?: Record<string, unknown>) => Promise<void>): Promise<boolean>;
+    setStatusChangeMap(name: string, statusChange: (status: string, token?: string, accountInfo?: Record<string, unknown>) => Promise<void>, immediaCall?: boolean): Promise<boolean>;
```
Comment (swatDong) [inline packages/api/src/utils/login.ts:55]: use the full word... immedia**te**Call

File: packages/api/src/utils/login.ts [hunk 2]
```diff
@@ -50,8 +50,9 @@ export interface AzureAccountProvider {
      * Add update account info callback 
      * @param name callback name
      * @param statusChange callback method
+     * @param immediaCall whether callback when register
```
Comment (swatDong) [inline packages/api/src/utils/login.ts:53]: callout that default is true

Comment (xiaolang124) [reply packages/api/src/utils/login.ts]: Fixed.

**Review by swatDong — APPROVED** [review]

**Pull request merged**